package/MDAnalysis/coordinates/timestep.pyx

```diff
@@ -493,20 +493,31 @@ cdef class Timestep:
         vectors lengths followed by their respective angle, or as three
         triclinic vectors.
 
-          >>> ts.dimensions
-          array([ 13.,  14.,  15.,  90.,  90.,  90.], dtype=float32)
-          >>> ts.triclinic_dimensions
-          array([[ 13.,   0.,   0.],
-                 [  0.,  14.,   0.],
-                 [  0.,   0.,  15.]], dtype=float32)
+          .. testsetup::
+
+          >>> import MDAnalysis as mda
+          >>> from MDAnalysis.tests.datafiles import TPR, XTC
+          >>> import numpy as np
+          >>> u = mda.Universe(TPR, XTC)
+          >>> ts = u.trajectory[0] 
+               
+          .. doctest::
+                    
+          >>> print(np.round(ts.dimensions))
```

Comment:
Is rounding really required? It makes for less clear documentation. I'd like to get rid of it if at all possible so that it does not distract from what we are really interested in, namely `ts.dimensions`. The documentation aspect is more important than the testing...